nifi-nar-bundles/nifi-elasticsearch-bundle/nifi-elasticsearch-client-service/src/main/java/org/apache/nifi/elasticsearch/ElasticSearchClientServiceImpl.java

```diff
@@ -61,22 +58,23 @@
 public class ElasticSearchClientServiceImpl extends AbstractControllerService implements ElasticSearchClientService {
     private ObjectMapper mapper = new ObjectMapper();
 
-    private List<PropertyDescriptor> properties;
+    static final private List<PropertyDescriptor> properties;
 
     private RestClient client;
 
     private String url;
 
-    @Override
-    protected void init(ControllerServiceInitializationContext config) {
-        properties = new ArrayList<>();
-        properties.add(ElasticSearchClientService.HTTP_HOSTS);
-        properties.add(ElasticSearchClientService.USERNAME);
-        properties.add(ElasticSearchClientService.PASSWORD);
-        properties.add(ElasticSearchClientService.PROP_SSL_CONTEXT_SERVICE);
-        properties.add(ElasticSearchClientService.CONNECT_TIMEOUT);
-        properties.add(ElasticSearchClientService.SOCKET_TIMEOUT);
-        properties.add(ElasticSearchClientService.RETRY_TIMEOUT);
+    static {
+        List _props = new ArrayList();
```

Comment:
Bit of a nit pick but can this init be changed so it's not of raw type:

> List\<PropertyDescriptor\> _props = new ArrayList\<\>();

Reply:
Done.